k8s/demo/common/ccd/ccd-logstash.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccd-logstash
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: "true"
+spec:
+  releaseName: ccd-logstash
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:elastic/helm-charts
+    ref: master
+    path: tree/master/logstash
+  values:
+    image: "hmctspublic.azurecr.io/imported/logstash"
+    imageTag: "6.8.13"
+    imagePullPolicy: "IfNotPresent"
+    logstashConfig:
+      logstash.yml: |
+        xpack.monitoring:
+          enabled: true
+        xpack.monitoring.elasticsearch:
+          url: ["http://ccd-data-0.service.core-compute-demo.internal:9200","http://ccd-data-1.service.core-compute-demo.internal:9200","http://ccd-data-2.service.core-compute-demo.internal:9200"]
+    logstashPipeline:
+      logstash.conf: |
+        input {
+          jdbc {
+            jdbc_connection_string => "${DB_URL}"
```

Comment:
when are these placeholders replaced?

Reply:
I just need to set them as env vars, but need to grab the env vars from vault... will all be done shortly enough